Formula/lua.rb

```diff
@@ -45,7 +45,8 @@ class Lua < Formula
   # sigaction provided by posix signalling power patch
   if build.with? "sigaction"
     patch do
-      url "http://lua-users.org/files/wiki_insecure/power_patches/5.2/lua-5.2.3-sig_catch.patch"
+      # url "http://lua-users.org/files/wiki_insecure/power_patches/5.2/lua-5.2.3-sig_catch.patch"
+      url "https://gist.githubusercontent.com/anonymous/99614c27bcac2aa86181fabe78c85915/raw/7abdf0234ffd38fc9094078d8128938f5e2accd4/lua-5.2.3-sig_catch.patch"
```

Comment:
I apologize if my comments sounded dismissive. They were not intended to, so I'm blaming this on the medium (text) and my inability to express my thoughts adequately. My initial comment was sparked by genuine curiosity as to how the rename would help and the conviction that an already well established repository shouldn't be renamed lightly, even if the repository in question is probably among the least critical to Homebrew and mostly invisible to its users and contributors.

(Just wanted to hopefully clarify a few things. I'm not expecting to see a response and not at all intending to re-spark the discussion.)